docs/91_version_history.rst

```diff
@@ -25,6 +25,152 @@ Version History
+       - The foglamp-filter-expression filter has been modified to better deal with streams which contain multiple assets. It is now possible to use the syntax <assetName>.<datapointName> in an expression in addition to the previous <datapointName>. The result is that if two assets in the data stream have the same data point names it is now possible to differentiate between them.
+       - A new plugin to collect variables from Beckhoff PLC's has been written. The plugin uses the TwinCAT 2 or TwinCAT 3 protocols to collect specified variable from the running PLC.
+
+
+    - Bug Fix:
+
+       - An issue in the sending of data to the PI server with large values has been resolved.
+       - The playback south plugin was not correctly replaying timestamps within the file, this has now been resolved.
+       - Use of the asset filter in a north task could result in the north task terminating. This has now resolved.
+       - A small memory leak in the south service statistics handling code was impacting the performance of the south service, this is now resolved.
+       - An issue has been discovered in the Fair camera plugin with the validity attribute of the spot temperatures, this has now been resolved.
+       - It was not possible to send data for the same asset from two different FogLAMP’s into the PI Server using PI Web API, this has now been resolved.
+       - The filter FogLAMP RMS Trigger was not able to be dynamically reconfigured, this has now been resolved.
+       - If a filter in the north sending process increased the number of readings it was possible that the limit of the number of readings sent in a single block . The sending process will now ensure this can not happen.
+       - RMS filter plugin was not able to be dynamically reconfigured, this has now been resolved.
+       - The HTTP South plugin that is used to receive data from another FogLAMP instance may fail with some combinations of filters applied to the service. This issue has now been resolved.
+       - The rule filter may give errors if expressions have variables not satisfied in the reading data. Under some circumstances it has been seen that the filter fails to process data after giving this error. This has been resolved by changes to make the rate filter more robust.
+       - Blank values for asset names in the south service may cause the service to become unresponsive. Blank asset names have now been correctly detected, asset names are required configuration values.
+       - A new version of the driver software for the USB-4704 Data Acquisition Module has been released, the plugin has been updated to use this driver version.
+       - The OPC UA North plugin might report incorrect counts for sent readings on some platforms, this has now been resolved.
+       - The simple Python filter plugin was not adding correct asset tracking data, this has now been updated.
```

Comment:
N/A for fledge release

Reply:
Unfortunately we have no way to remove this, so will have to live with it for 1.8.

Reply:
okay